lib/chef/provider/user/dscl.rb

```diff
@@ -329,7 +329,7 @@ def ditto_home
 
         def move_home
           Chef::Log.debug("#{new_resource} moving #{self} home from #{current_resource.home} to #{new_resource.home}")
-
+          new_resource.gid(20) if new_resource.gid.nil?
```

Comment:
Would be really nice if the user resource natively supported the osx IsHidden property too.